pkg/resources/resource_monitor_acceptance_test.go

```diff
@@ -338,6 +344,154 @@ func TestAcc_ResourceMonitor_Updates(t *testing.T) {
 	})
 }
 
+func TestAcc_ResourceMonitor_ExternalChanges(t *testing.T) {
+	id := acc.TestClient().Ids.RandomAccountObjectIdentifier()
+
+	configModelEverythingSet := model.ResourceMonitor("test", id.Name()).
+		WithNotifyUsersValue(configvariable.SetVariable(configvariable.StringVariable("JAN_CIESLAK"))).
+		WithCreditQuota(10).
+		WithFrequency(string(sdk.FrequencyWeekly)).
+		WithStartTimestamp(time.Now().Add(time.Hour * 24 * 30).Format("2006-01-02 15:01")).
+		WithEndTimestamp(time.Now().Add(time.Hour * 24 * 60).Format("2006-01-02 15:01")).
+		WithNotifyTriggersValue(configvariable.SetVariable(
+			configvariable.IntegerVariable(100),
+			configvariable.IntegerVariable(110),
+		)).
+		WithSuspendTrigger(120).
+		WithSuspendImmediateTrigger(150)
+
+	updatedStartTimestamp := time.Now().Add(time.Hour * 24 * 40).Format("2006-01-02 15:01")
+	updatedEndTimestamp := time.Now().Add(time.Hour * 24 * 70).Format("2006-01-02 15:01")
+	configModelUpdated := model.ResourceMonitor("test", id.Name()).
+		WithNotifyUsersValue(configvariable.SetVariable(configvariable.StringVariable("JAN_CIESLAK"), configvariable.StringVariable("ARTUR_SAWICKI"))).
+		WithCreditQuota(20).
+		WithFrequency(string(sdk.FrequencyMonthly)).
+		WithStartTimestamp(updatedStartTimestamp).
+		WithEndTimestamp(updatedEndTimestamp).
+		WithNotifyTriggersValue(configvariable.SetVariable(
+			configvariable.IntegerVariable(110),
+			configvariable.IntegerVariable(120),
+		)).
+		WithSuspendTrigger(130).
+		WithSuspendImmediateTrigger(160)
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
+		PreCheck:                 func() { acc.TestAccPreCheck(t) },
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.RequireAbove(tfversion.Version1_5_0),
+		},
+		CheckDestroy: acc.CheckDestroy(t, resources.ResourceMonitor),
+		Steps: []resource.TestStep{
+			{
+				Config: config.FromModel(t, configModelEverythingSet),
+			},
+			// Update externally, but match the updated configuration (expected updates to the same values)
```

Comment:
hmmm, but why we expect changes if we have ignorechange to current snowflake value? (e.g. check user_acceptance_test `CHANGE PROP TO THE CURRENT SNOWFLAKE VALUE` in `TestAcc_User_BasicFlows`). Isn't the triggers collection causing the update here? Can we check what happens when we only alter the basic attributes?

Reply:
From the printed plan I saw that `notify_triggers` and `notify_users` are computed (because we have no diff suppression for them) causing `show_output` to have changes. But even when I changed them to be the same in step 1 and 2 (and didn't change them externally) they were still computed causing `show_output` to change and produce the plan. Ultimately, I wanted to put ExpectNoPlan() plan check, but couldn't because of that (I think).